Card page revamp

abigaeljamie opened the pull request:
Closes [issue 1786](https://github.com/patternfly/patternfly-org/issues/1786)
Closes [issue 1804](https://github.com/patternfly/patternfly-org/issues/1804)

patternfly-build commented:
PF3 preview: https://patternfly-org-pr-1924-v3.surge.sh/v3

mcarrano commented on packages/v4/src/content/design-guidelines/usage-and-behavior/card-view/cards.md:
```diff
@@ -0,0 +1,138 @@
+<img src="./img/dashboard.png"  alt="card view with charts"  width="990"/> 
+
+Do NOT use a card:
+- Behind a component, like a data table or a list, when it's the only section on the page. (But when there are both cards and tables on the page, you can separate the page using cards.) 
```
Can we strike the first 2 bullets here and the screen shot below? Due to some implementation issues, we are rethinking changes to card usage for full screen views.

abigaeljamie replied:
Thanks, @mcarrano ! I removed those bullet items and the screenshot.

mmenestr commented on packages/v4/src/content/design-guidelines/usage-and-behavior/card-view/cards.md:
```diff
@@ -0,0 +1,135 @@
+1. **Compact (recommended)**: Compact cards use 16px spacing with an 8px spacer between the header and body and 16px between the body and footer.
+2. **Large**: Large cards use 24px spacing with a 16px spacer between the header and body and 24px between the body and footer.  
+
+<img src="./img/card-types.png"  alt="compact and standard card types"  width="779"/> 
```
Found a small inconsistency here. In the text it's referred to as "Large" but in the image I referred to it as "Standard" card. @mcarrano which one is the correct one. If it's large let me know so I can update the image! :)

mcarrano replied:
Good question. I think we settled on "Large." Can you confirm @gdoyle1 ? But I think it sounds wrong to say "Large card title because the size of the card title is not different. Can we update to just call this a "Card title" in either case?

mmenestr replied:
It seems like in pf org documentation it's called "Basic". Also, I can definitely update the card titles to just say "Card title"

gdoyle1 replied:
Sorry for the delayed response @mcarrano @mmenestr @abigaeljamie - Margot and I settled on:
1. **Compact (recommended)**: Compact cards use 16px spacing with an 8px spacer between the header and body and 16px between the body and footer.
2. **Standard**: Standard cards use 24px spacing with a 16px spacer between the header and body and 24px between the body and footer.  

We spoke with Matt Nolting and found that in code, you will get the "standard" spacing by default and only will receive the "compact" spacing if you apply the` pf-m-compact`. modifier. So I think that makes the most sense! I will update in sketch library too.

abigaeljamie commented:
Will do, @mmenestr ! Can you please email me those images so that I have all of them handy?

abigaeljamie commented:
Thanks for the feedback! I replaced the copy so that we have the Compact and Standard card types, and I replaced all the images from @mmenestr . The only thing I need is one more image (it appears broken in the preview because I deleted it). @gdoyle1 , do you mind sending it to me again? You sent it to me a few weeks back in Slack, but Slack erased it. It's the image that shows the sticky bottom toolbar.
